marketdata/market/dex/dex.go

```diff
@@ -20,13 +20,14 @@ type Market struct {
 }
 
 func InitMarket() market.Provider {
+	update := viper.GetDuration("market.dex_quote_update_seconds")
 	m := &Market{
 		Market: market.Market{
 			Id:         "dex",
 			Name:       "Binance Dex",
```

Comment:
is name required param?

Reply:
no, name and url it's optional. Id and API it's required